content/en/load-testing/Quick Start/20-demo-application.md

```diff
@@ -8,7 +8,7 @@ description: >
   How to get and run Posters, the demo application.
 ---
 
-All official XLT releases ship with a real-world demo web application (*Posters*) as the system under test and a [test suite](../30-demo-test-suite) to test this application. Both can be found in the directory `<XLT>/samples`.
+All official XLT releases ship with a real-world demo web application (*Posters*) as the system under test. It can be found in the directory `<XLT>/samples`.
```

Comment:
All XLT releases are shipped with the "Posters" demo application. Posters is a small e-commerce store and is used to learn and demo XLT.